modules/gapi/include/opencv2/gapi/streaming/onevpl/accel_types.hpp

```diff
@@ -20,6 +20,7 @@ namespace onevpl {
 enum class AccelType: uint8_t {
     HOST,
     DX11,
+    VA_API,
```

Comment:
or VAAPI? That's how it's called at 01.org - https://01.org/linuxmedia/vaapi

Reply:
you right, i'll rename it